src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngineLike.scala

```diff
@@ -17,7 +17,7 @@ package markets.clearing.engines
 
 import markets.clearing.strategies.PriceFormationStrategy
 import markets.orders.{AskOrder, BidOrder, Order}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1428464756/issues/source?bid=2921361&fileBranchId=2988596#l19)